Dockerfile.test

```diff
@@ -0,0 +1,26 @@
+FROM 1science/alpine
+
+ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+# get our basic-needs sorted
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories
```

Comment:
Can we fetch via https instead of http?  Is there a security issue here?

Reply:
I didn't catch this.  I made the change locally, but if you look at the output of `apk update`:

```
Step 4/12 : RUN apk update
 ---> Running in 1d4d39cd1d7f
fetch http://dl-cdn.alpinelinux.org/alpine/v3.5/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.5/community/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
```
Notice that the main package indices are downloaded with SSL protection.  I searched the Web, this is a known issue (gliderlabs/docker-alpine#184).

In addition, it looks like they have a bad certificate:

```
tim77code-limbo raymie$ curl -v https://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
*   Trying 151.101.0.249...
* Connected to dl-cdn.alpinelinux.org (151.101.0.249) port 443 (#0)
* SSL certificate problem: Invalid certificate chain
* Closing connection 0
curl: (60) SSL certificate problem: Invalid certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
```
I could apply `sed` to the package index to convert all URLs to SSL, but I'm now wondering if Alpine is secure.  I got everything working on Ubuntu over the weekend, should I switch over to that?

Reply:
I'd prefer to use Ubuntu, if that's approximately the same amount of work.

Reply:
I left the http:// URL in place.  Instead, let's switch to Ubuntu.  I opened a ticket for the switch (#6).